### DISCUSSION TITLE
How can I use pygwalker in marimo?

### DISCUSSION THREAD

**Discussion by Dumbliidore** [marimo-team/marimo#1756] [Q&A] [answered]
There is a warning when I use pygwalker in marimo

![image](https://github.com/marimo-team/marimo/assets/35067857/e56002c0-22dd-4752-83cc-26a811791dd8)

this is the code
```python
import pygwalker as pyg
import pandas as pd

df= pd.read_csv("./movie.csv")
walker = pyg.walk(df, kernel_computation=True)
```

and then I configure the max output size with the environment variable MARIMO_OUTPUT_MAX_BYTES.
```python
import os

os.environ['MARIMO_OUTPUT_MAX_BYTES'] = '1000_000_000'
```

it still doesn't work.

**Comment by mscolnick** [comment, marked as answer]
You'll need to do `export MARIMO_OUTPUT_MAX_BYTES=1000000000` in your terminal since it uses the output at startup time

**Reply by Dumbliidore** [reply]
Thanks, it's working.
